concept/thing/thing_manager.rs

```diff
@@ -987,6 +983,8 @@ impl<'txn> ThingManager {
                                     });
                                 }
                             }
+                            AttributeTypeAnnotation::Range(_) => todo!("create attribute with range"),
+                            AttributeTypeAnnotation::Values(_) => todo!("create attribute with values"),
```

Comment:
Let's make sure we implement these data validations soon!

Reply:
On my way!